templates/patterns/button/pattern-button.html.twig

```diff
@@ -4,11 +4,35 @@
  * Button.
  */
 #}
-{% include '@ecl/ec-component-button' with {
-  'modifier': variant|replace({'_':'-'}),
-  'type_attribute': type_attribute,
-  'href': href,
-  'icon': icon,
-  'label': label,
-  'is_block': is_block
-} only %}
+{% if variant == 'default' %}
+  {% include '@ecl/link' with {
+    'link': {
+      'type': type_attribute,
+      'label': label,
+      'path': href,
+      'icon_position': icon_position
```

Comment:
Default it to `after`, if possible.

Reply:
In the twig template, the icon_position have already 'after' as default.
`{% set _icon_position = icon_position|default('after') %}`

But we could put here too.